packages/wrangler/src/type-generation/helpers.ts

```diff
@@ -1,10 +1,14 @@
-import { readFileSync } from "node:fs";
+import { readFileSync, writeFileSync } from "node:fs";
 import { version } from "workerd";
 import { logger } from "../logger";
+import { generateRuntimeTypes } from "./runtime";
 import { generateEnvTypes } from ".";
 import type { Entry } from "../deployment-bundle/entry";
 import type { Config } from "@cloudflare/workers-utils";
 
+export const DEFAULT_WORKERS_TYPES_FILE_NAME = "worker-configuration.d.ts";
+export const DEFAULT_WORKERS_TYPES_FILE_PATH = `./${DEFAULT_WORKERS_TYPES_FILE_NAME}`;
```

Comment:
I don't think two variables here are necessary, this seems to work perfectly fine:
<img width="803" height="77" alt="Screenshot 2025-12-18 at 11 59 04" src="https://github.com/user-attachments/assets/271a8b87-db1a-46e7-bcbb-cb16ada5e29d" />

Reply:
If you prefer it this way I'm ok with it 👍